### ISSUE TITLE
How can I make only new or due cards show up in a filtered deck?

### ISSUE THREAD

**Issue by z7126z** [sachingooo/anki-filtered-deck-by-tag#4]
Hello! I'm sorry if this is question has been answered previously, but I'm struggling to find the solution. 

The way I currently use this add on is by watching a video over a certain topic, creating a filtered deck by the tag that corresponds to the aforementioned video, studying the entire filtered deck to get my cards into the learning state, and then deleting the filtered deck so that way I can study everything randomly by accessing my primary parent deck. 

Here's my issue though. Some of my cards that share the same tag are new, while some cards have been studied before and have a due date **far** into the future (which is a good thing for me!!). However, whenever I create these filtered decks, it pulls ALL of the cards in and makes me study the both the new cards and the cards that I have **already done** as if they are due **today**, when in reality they are not due for weeks... which ultimately pushes their due date back even further and essentially tampers with the card's learning algorithm.

Is there a solution for this? I absolutely love this add on but I am struggling to find a way to get only new cards and cards that are due today to show up. 

Thank you in advance for your help!

**Comment by sachingooo** [comment]
Hello! 

Have you tried setting one of the supplementalSearchTexts entries to be "is:due OR is:new" ?

**Comment by z7126z** [comment]
Yes! I saw that solution was posted in another thread so I tried that, but whenever I did, nothing changed. I figured I'd need to restart Anki to implement the changes, and whenever I did that and returned to the browser menu to right click the tab and create the new filtered deck, the option to create the deck was no longer there. It was quite puzzling.

**Comment by sachingooo** [comment]
Can you copy and paste what your current config looks like for this addon?

**Comment by z7126z** [comment]
Of course. First I will display here what it looks like now, and then I will display what the configuration looked like when I tried the previous solution.

{
    "defaultOrder": 6,
    "numCards": 9999,
    "shortNames": [],
    "supplementalSearchTexts": [],
    "unsuspendAutomatically": true
}

{
    "defaultOrder": 6,
    "numCards": 9999,
    "shortNames": [],
    "supplementalSearchTexts": [
        "(is:due OR is:new)"
    ],
    "unsuspendAutomatically": true
}

**Comment by z7126z** [comment]
Hello! Were you able to find a solution to this?
